examples/motorkit_dc_test.py

```diff
@@ -17,7 +17,7 @@
         time.sleep(0.01)
 
     print("Slow down...")
-    for i in reversed(range(0, 100)):
+    for i in range(100, 0, -1)):
```

Comment:
This does not appear to function the same as the previous version, at least based on checking in the REPL. 

Current:
```
>>> for i in reversed(range(0, 10)):
...     print(i)
...
9
8
7
6
5
4
3
2
1
0
```
Proposed change:
```
>>> for i in range(10, 0, -1):
...     print(i)
...
10
9
8
7
6
5
4
3
2
1
```
What appears to work like the current version:
```
>>> for i in range(9, -1, -1):
...     print(i)
...
9
8
7
6
5
4
3
2
1
0
>>>
```

Reply:
Good point. The statement should be corrected to
`for i in range(100, -1, -1)):` 
so that the counter value ranges from 100 to 0, resulting in a calculated throttle value range of 1.0 to 0, from full speed to stopped.
The original started with a calculated throttle value of 0.9.

revised:
![image](https://user-images.githubusercontent.com/29906257/52905784-04927880-31f4-11e9-8f4c-eb7b8d179f82.png)